src/pages/Entries/Entries.tsx

```diff
@@ -15,26 +18,72 @@ export default function Entries() {
   const [editing, setEditing] = useState(false);
   const prevSelectedEntryId = useRef<number | undefined>(undefined);
   const { user } = useUser();
+  const { api } = useApi();
+  const queryClient = useQueryClient();
```

Comment:
shouldn't need this? (its in the api context)

Reply:
I tried doing what you said to get query client from the `useApi()` call, but I could not do it since I'm pretty sure React query requires that you use their `QueryClientProvider` component and the `useQuery()` hook. If you can get it working that way feel free but I don't think it's doable from my testing.